src/main/java/Doot.java

```diff
@@ -0,0 +1,149 @@
+import java.util.Scanner;
+
+public class Doot {
+    private static final String DIVIDER = "____________________________________________________________\n\n";
+    private static final int MAX_TASKS = 100;
+    private static Task[] taskList = new Task[MAX_TASKS];
+    private static int taskIdx = 0;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print(DIVIDER + "Hello! I'm  Doot\nWhat can I do for you?\n" + DIVIDER);
+        String currentInput = scanner.nextLine();
+        while (!currentInput.equals("bye")) {
+            findCommand(currentInput);
+            currentInput = scanner.nextLine();
+        }
+        System.out.print(DIVIDER + "Bye. Hope to see you again soon!" + "\n" + DIVIDER);
+        scanner.close();
+    }
+
+    public static void findCommand(String command) {
+        String wordDigit = "\\d{1,3}$";
+        String deadlineMatch = "(\\w+ )*/by .+";
+        String eventMatch = "(\\w+ )*/from (\\w+ )*/to .+";
+        boolean containsSpace = command.contains(" ");
+        String cmd = containsSpace ? command.substring(0, command.indexOf(" ")) : command;
+        String args = containsSpace ? command.substring(command.indexOf(" ") + 1) : "";
+        if (args.matches(wordDigit)) {
+            handleWordDigit(cmd, args);
+        } else if (args.matches(eventMatch)) {
+            handleEvent(cmd, args);
+        } else if (args.matches(deadlineMatch)) {
+            handleDeadline(cmd, args);
+        } else {
+            handleDefault(cmd, args);
+        }
+    }
+
+    private static void handleWordDigit(String command, String args) {
+        int digit = Integer.parseInt(args);
+        switch (command) {
+            case "mark":
+                markTask(digit);
+                break;
+            case "unmark":
+                unmarkTask(digit);
+                break;
+            default:
+                addToList(command);
+                break;
```

Comment:
No indentation required for "case" statements, do look for similar issues below in the code